docs/source/user-guide/configuration/configuration-settings.rst

```diff
@@ -0,0 +1,807 @@
+
+.. _specify-env-directories:
+
+Specify environment directories (envs_dirs)
+-------------------------------------------
+
+Specify directories in which environments are located. If this
+key is set, the root prefix ``envs_dir`` is not used unless
+explicitly included. This key also determines where the package
+caches are located.
+
+For each envs here, ``envs/pkgs`` is used as the pkgs cache,
+except for the standard ``envs`` directory in the root
+directory, for which the normal ``root_dir/pkgs`` is used.
+
+**Example:**
+
+.. code-block:: yaml
+
+  envs_dirs:
+    - ~/my-envs
+    - /opt/anaconda/envs
+
+The ``CONDA_ENVS_PATH`` environment variable overwrites the ``envs_dirs`` setting:
+
+* For macOS and Linux:
+  ``CONDA_ENVS_PATH=~/my-envs:/opt/anaconda/envs``
+
+* For Windows:
+  ``set CONDA_ENVS_PATH=C:\Users\joe\envs;C:\Anaconda\envs``
+
+.. _specify-pkg-directories:
+
+Specify package directories (pkgs_dirs)
+---------------------------------------
+
+Specify directories in which packages are located. If this
+key is set, the root prefix ``pkgs_dirs`` is not used unless
+explicitly included.
+
+If the ``pkgs_dirs`` key is not set, then ``envs/pkgs`` is used
+as the pkgs cache, except for the standard ``envs`` directory in the root
+directory, for which the normal ``root_dir/pkgs`` is used.
+
+**Example:**
+
+.. code-block:: yaml
+
+  pkgs_dirs:
+    - /opt/anaconda/pkgs
+
+The ``CONDA_PKGS_DIRS`` environment variable overwrites the
+``pkgs_dirs`` setting:
+
+* For macOS and Linux:
+  ``CONDA_PKGS_DIRS=/opt/anaconda/pkgs``
+
+* For Windows:
+  ``set CONDA_PKGS_DIRS=C:\Anaconda\pkgs``
+
+.. _use-only-tar-bz2:
+
+Force conda to download only .tar.bz2 packages (use_only_tar_bz2)
+-----------------------------------------------------------------
+
+Conda 4.7 introduced a new ``.conda`` package file format.
+``.conda`` is a more compact and faster alternative to ``.tar.bz2`` packages.
+It's thus the preferred file format to use where available.
+
+Nevertheless, it's possible to force conda to only download ``.tar.bz2`` packages
+by setting the ``use_only_tar_bz2`` boolean to ``True``.
+
+The default is ``False``.
+
+**Example:**
+
+.. code-block:: yaml
+
+  use_only_tar_bz2: True
+
+.. note::
+
+   This is forced to ``True`` if conda-build is installed and older than 3.18.3,
+   because older versions of conda break when conda feeds it the new file format.
+
+Conda-build configuration
+=========================
+
+.. _specify-root-dir:
+
+Specify conda-build output root directory (root-dir)
+----------------------------------------------------
+
+Build output root directory. You can also set this with the
+``CONDA_BLD_PATH`` environment variable. The default is
+``<CONDA_PREFIX>/conda-bld/``. If you do not have write
+permissions to ``<CONDA_PREFIX>/conda-bld/``, the default is
+``~/conda-bld/``.
+
+**Example:**
+
+.. code-block:: yaml
+
+  conda-build:
+      root-dir: ~/conda-builds
+.. _specify-output-folder:
```

Comment:
Could you please check more of those markers that there is a linebreak before them?